src/sentry/services/hybrid_cloud/notifications/serial.py

```diff
@@ -13,3 +14,23 @@ def serialize_external_actor(actor: ExternalActor) -> RpcExternalActor:
         external_name=actor.external_name,
         external_id=actor.external_id,
     )
+
+
+def serialize_group_subscription_status(
+    status: GroupSubscriptionStatus,
+) -> RpcGroupSubscriptionStatus:
+    return RpcGroupSubscriptionStatus(
+        is_disabled=status.is_disabled,
+        is_active=status.is_active,
+        has_only_inactive_subscriptions=status.has_only_inactive_subscriptions,
+    )
+
+
+def deserialize_group_subscription_status(
+    status: RpcGroupSubscriptionStatus,
+) -> GroupSubscriptionStatus:
+    return GroupSubscriptionStatus(
+        is_disabled=status.is_disabled,
+        is_active=status.is_active,
+        has_only_inactive_subscriptions=status.has_only_inactive_subscriptions,
+    )
```

Comment:
It feels a little ridiculous to define a new class (`RpcGroupSubscriptionStatus`) that maps so exactly onto an existing dataclass. But the alternative would be to make `GroupSubscriptionStatus` extend `RpcModel` when its other uses have nothing to do with RPCs, so I suppose it's worth it.

Reply:
Having the extra dataclass for RPC boundaries is preferrable for me. It helps isolate RPC interfaces from the application domain models.